simulation/src/simeeprom.cpp

```diff
@@ -22,15 +24,98 @@ void  EEPROMClass::write( int idx, uint8_t val )
 {
     printf("Writing to EEPROM address %u = %u\n", idx, val);
     someFakeEEPROM_memory[idx] = val;
+
+    if(autoUpdate && storage)
+    {
+        fseek(storage, idx, SEEK_SET);
+        fputc(val, storage);
+        fflush(storage);
+    }
 }
 
 void  EEPROMClass::update( int idx, uint8_t val )
 {
     write(idx, val);
```

Comment:
Update should check if the value in EEPROM actually changed. I didn't care to implement this before, but now that the data can be saved on changes, it might be a good idea to add it.

Reply:
Now it does!